mlir/test/Dialect/Linalg/hoisting.mlir

```diff
@@ -308,6 +308,61 @@ module attributes {transform.with_named_sequence} {
 
 // -----
 
+// CHECK-LABEL:  func.func @no_hoisting_zero_trip_loop
+func.func @no_hoisting_zero_trip_loop(%arg0: memref<20xi32>, %lb: index, %ub: index) {
```

Comment:
For consistency with the rest of the file. 
```suggestion
func.func @no_hoisting_zero_trip_loop(%memref0: memref<20xi32>, %lb: index, %ub: index) {
```

IMHO, `%memref0` is a bit ambiguouis/unfortunate and I'd be tempted to update everything to e.g. `%src`.